nomad/status_endpoint.go

```diff
@@ -22,7 +22,11 @@ func NewStatusEndpoint(srv *Server, ctx *RPCContext) *Status {
 }
 
 // Ping is used to just check for connectivity
-func (s *Status) Ping(args struct{}, reply *struct{}) error {
+func (s *Status) Ping(args structs.GenericRequest, reply *struct{}) error {
+	// note: we're intentionally throwing away any auth error here
+	identity, _ := s.srv.Authenticate(s.ctx, args.AuthToken)
+	args.SetIdentity(identity)
```

Comment:
Why do we call this if we don't use either of the results? Might want to update the comment to explain why it's called at all instead of just pointing out that the `_` is intentional